test/mlir/onnx/parse/test_abs.json

```diff
@@ -0,0 +1,78 @@
+// RUN: sed 's/^[ \t]*\/\/.*//' %s | onnx-mlir --EmitONNXBasic --printIR | FileCheck %s
```

Comment:
Maybe it was explained elsewhere, would it be possible to hide the sed within the onnx-mlir pipeline? This feels cumbersome to use.

Reply:
yes, we can remove // comments in onnx-mlir, then we don't need to strip the comments with sed in the lit tests, as I mentioned here:
https://github.com/onnx/onnx-mlir/pull/1685#discussion_r966182472
https://github.com/onnx/onnx-mlir/pull/1685#discussion_r966179380

I went ahead and did that, and then we don't need to read input from stdin anymore